src/full-monero-node-external-sd-install.sh

```diff
@@ -43,23 +45,104 @@ termux-wake-lock -y
 pkg update -y
 pkg install nano wget termux-api jq -y
 
-# Pre-Clean Old Setup
-rm -f $TERMUX_BOOT/before_start_monero_node
-rm -f $TERMUX_BOOT/*XMR\ Node*
-
 # Create Directories
+
 mkdir -p $MONERO_CLI
-mkdir -p $NODE_DATA
+mkdir -p $NODE_CONFIG
 mkdir -p $TERMUX_BOOT
 mkdir -p $TERMUX_SHORTCUTS
 mkdir -p $TERMUX_SCHEDULED
-mkdir -p $NODE_CONFIG
 
+# Pre-Clean Old Setup
+rm -f $TERMUX_BOOT/before_start_monero_node
+rm -f $TERMUX_BOOT/*XMR\ Node*
+NODE_CONFIGOLD=~/monero-cli/config
+if [ -d $NODE_CONFIGOLD ]
+then
+mv $NODE_CONFIGOLD/* $NODE_CONFIG/
+rm -r $NODE_CONFIGOLD
+fi
 
-# Download Blocklist (disabled)
-cd $NODE_CONFIG
-# wget -O block.txt https://gui.xmr.pm/files/block.txt
+SD=~/storage/external-1
```

Comment:
Good spot for a comment denoting what's happening in this section... i.e. something like `# Set blockchain location - internal or external SD`

Reply:
The comment is 2 lines below. Feel free to move

Reply:
Moved